cpp/ql/test/query-tests/jsf/4.13 Functions/AV Rule 114/test.cpp

```diff
@@ -48,7 +48,7 @@ MyValue g7(bool c)
 	if (c) return MyValue(7);
 	DONOTHING
 	DONOTHING
-	// BAD [the alert here is unfortunately placed]
+	// GOOD
```

Comment:
Shouldn't this be `// BAD` - i.e. it is correct to produce an alert here?

Reply:
Good point.  Corrected.